pkg/policy/repository.go

```diff
@@ -26,12 +26,16 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ruleSlice is a wrapper around a slice of *rule, which allows for functions
+// to be written with []*rule as a receiver.
+type ruleSlice []*rule
```

Comment:
Similarly to the `pkg/api/rules.go` create a `pkg/policy/rules.go`

Reply:
Done.